src/js/components/dashboard/graph/active/SignificanceGraph.jsx

```diff
@@ -95,10 +96,21 @@ export default class SignificanceGraph extends React.Component {
 
         // have an inverted range so that the yScale output returns the correct Y position within
         // the SVG element (y = 0 is the top of the graph)
-        values.yScale = scaleLinear()
-            .domain([0, yMax])
-            .range([values.graphHeight, 0])
-            .clamp(true);
+        if (yMax < 1000) {
+            values.yScale = scaleLinear()
+                .domain([0, yMax])
+                .range([values.graphHeight, 0])
+                .clamp(true);
+        }
+        else {
+            this.setState({ logScale: true });
+            values.yScale = scaleLog()
+                .domain([1, yMax])
```

Comment:
It's weird that it has to start with 1 but it makes sense given that starting from 0 would break the scaling.